docs/source/technical_features.md

```diff
@@ -37,10 +60,11 @@ the prior knowledge about the subjective lesion borders is then leveraged to the
 ![Figure Data Augmentation on lesion ground truths](https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/dilate-gt.png)
 
 ## Network architectures
-- [U-net](https://arxiv.org/pdf/1505.04597.pdf)
-- [HeMIS](https://arxiv.org/abs/1607.05194)
+- [UNet](https://arxiv.org/pdf/1505.04597.pdf), with control of the network depth.
+- HeMIS-UNet: integrates the [HeMIS](https://arxiv.org/abs/1607.05194) strategy to deal with missing modalities within a UNet training scheme.
+- FiLMed-UNet, based on [FiLM](https://arxiv.org/pdf/1709.07871.pdf) strategy adapted to the [segmentation task](#physic-informed-network).
 
 ## Loss functions
-- [Dice Loss](https://arxiv.org/abs/1606.04797)
-- [Focal Loss](https://arxiv.org/pdf/1708.02002.pdf)
-- [Generalised Dice Loss](https://arxiv.org/pdf/1707.03237.pdf)
+- [Dice Loss](https://arxiv.org/abs/1606.04797). Also adapted for multi-label segmentation tasks, by averaging the loss for each class.
+- [Focal Loss](https://arxiv.org/pdf/1708.02002.pdf). Additional possibility to be combined with the DiceLoss is a linear combination.
```

Comment:
I don't understand this sentence: "Additional possibility to be combined with the DiceLoss is a linear combination.". What does it have to do with the focal loss?
```suggestion
- [Focal Loss](https://arxiv.org/abs/1708.02002). Additional possibility to be combined with the DiceLoss is a linear combination.
```